providers/dns/cloudns/internal/client.go

```diff
@@ -268,7 +268,7 @@ func toUnreadableBodyMessage(req *http.Request, rawBody []byte) string {
 // 259200 = 3 days
 // 604800 = 1 week
 // 1209600 = 2 weeks
-// 2592000 = 1 month
+// 2592000 = 1 month.
```

Comment:
This whole comment should be reformatted.

Maybe so?

```go
// Rounds the given TTL in seconds to the next accepted value.
// Accepted TTL values are:
//  - 60 = 1 minute
//  - 300 = 5 minutes
//  - 900 = 15 minutes
//  - 1800 = 30 minutes
//  - 3600 = 1 hour
//  - 21600 = 6 hours
//  - 43200 = 12 hours
//  - 86400 = 1 day
//  - 172800 = 2 days
//  - 259200 = 3 days
//  - 604800 = 1 week
//  - 1209600 = 2 weeks
//  - 2592000 = 1 month
//  - 2592000 = 1 month
// See https://www.cloudns.net/wiki/article/58/ for details.
```

I believe `"//" + sp + sp + "-" + sp` is the magic marker for godoc to render a list item.